code/index.html

```diff
@@ -1,17 +1,208 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Business Site</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Picko — When you like to move it, move it</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
Your code is clear and clean and it was extremely hard to find anything to pick on but I had a try :'). Firstly, the inclusion of fonts using <link> tags is a really good idea and with the use of "preconnect" it should improve performance by reducing latency for external resources - very nice!